spec/controllers/settings_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe SettingsController, type: :controller do
+  render_views
+  let(:user) { FactoryGirl.create(:user) }
+
+  describe '#update_profile' do
+    let(:username) { 'miyukki' }
+    let(:password) { 'p@ssw0rd' }
+    let(:student_id) { 71349640 }
+    let(:ldap_bind_result) { true }
+    let(:ldap_info_result) {
+      double(:hash, {
+        uid: [username],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.